src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectResult = {};
+  const inputString = sourceString.split(';');
+  for (let i = 0; i < inputString.length; i += 1) {
```

Comment:
you can write just `i++`

Reply:
i know it, but airbnb and google recommend to write "i += 1"

Reply:
I agreed with you